### ISSUE TITLE
Intensive logging kills performance

### ISSUE THREAD

**Issue by JaapKeuter** [open744/terramaster#18]
The application is hammering the log files, due to the setting in terramaster.logging.properties
.level= FINEST
This slows down the application to a crawl. Hacking the jar and setting it to 'SEVERE' makes it run fine.
Request is to set it to a more appropriate level in the releases.

**Comment by Portree-Kid** [comment]
Look at Version 1.8. And Bugs towards my version shouldn't end up here. This seems dead

**Comment by wkitty42** [comment]
agreed on the life of this incidence of terramaster... not sure what's up with @open744 but they've been MIA for a while now...

i was very glad to see @Portree-Kid bring the project up to date... while i don't use it any more, i do still keep up with it... for now i'm using the terrasync.py that resides in FGMeta... i run it before every execution of fgfs and don't worry about new stuff that may be uploaded while i'm in the sim :smiley_cat: 

FWIW: about the only thing i'm currently aware of is that the SVN code in terramaster can be removed these days... the SVN stuff isn't being updated any more even though the directory structure is still being used and files are being added and updated...
